.github/workflows/test.yml

```diff
@@ -45,7 +47,7 @@ jobs:
     environment: integration-test-workflow
     strategy:
       matrix:
-        python-version: [ "3.6.15", "3.7", "3.8", "3.9", "3.10", "3.11" ]
+        python-version: [ "3.6.15", "3.11" ]
```

Comment:
Because I thought it was unnecessary -- no reason more than that. I don't expect that Python version should affect the Aerie interactions, but maybe that's wrong...

Reply:
I agree with that. Only thing I could think of is if the CLI's dependencies behavior differently between version.